lib/pubid/iso/typed_stage.rb

```diff
@@ -0,0 +1,43 @@
+module Pubid::Iso
+  class TypedStage
+    attr_accessor :type, :stage, :typed_stage
+
+    TYPED_STAGES = {
+      dtr: {
```

Comment:
In addition to the class of `TypedStage`, I'm thinking that it would be helpful to have: `Identifier::TechnicalReport`, `Identifier::InternationalStandard` that allows us to keep the relevant data together.